### ISSUE TITLE
Variations in Performances output are to big for use 

### ISSUE THREAD

**Issue by Lba-X** [GoogleChrome/lighthouse#9993]
I am currently working my way in to  Lighthouse 5.2. 

So far I am pretty content with what I have learned. But I seem unable to get a  stable output from the performance scan. As recommended I run five continuous scans and calculate the average. But the output vary a lot even though the conditions are the same.

In order to start using Lighthouse I need a stable output eg. for QA reporting and customer reviews.

In short my question is: How can I get an output from Lighthouse performance  that does not fluctuate more than eg. +/- 2 points

**Comment by patrickhulce** [comment]
Are you already applying the advice from our [various](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) [variability](https://docs.google.com/document/d/1BqtL-nG53rxWOI5RO0pItSRPowZVnYJ_gBEQCJ5EeUE/edit) [docs](https://docs.google.com/document/d/1AujmeKvBhzr-d8IsB7zPeS-vOtxCdw2GnspKpxJ7d_I/edit)?

+/- 2 points depending on where you fall on the scoring curve will likely require *all* the mitigations discussed across them.

**Comment by brendankenny** [comment]
There's been no movement on this issue, and the linked docs should be of help, so I'm going to close for now. If there's still an issue, feel free to comment and we can reopen!
